kahuna/public/js/edits/service.js

```diff
@@ -258,6 +258,13 @@ service.factory('editsService',
         var proposedMetadata = angular.copy(metadata);
         proposedMetadata[field] = value;
 
+        // peopleInImage is a special case. This turns a comma-separated string into an array trimmed of whitespace
```

Comment:
I haz no idea, but just thinking about users editing it sloppily (as we are giving them a simple text field for editing multiple entities): should it also be trimmed of commas?

Reply:
We could make it behave more like keywords. This is a bit of an in-between solution, where any tags are separated with a comma. I need to add more tests before this is ready for review.

Reply:
Absolutely good for it to go in like that IMHO regarding editing. Just was thinking of someone wanting to delete first entry in
`Tom Jerry, McDonald Trump`
and actually deleting *only* `Tom Jerry` leaving
`, McDonald Trump`.

Reply:
That’s handled. It would return an array containing only `McDonald Trump`.